src/backend/joanie/core/serializers/client.py

```diff
@@ -595,6 +620,20 @@ def get_total_currency(self, *args, **kwargs):
         return settings.DEFAULT_CURRENCY
 
 
+class OrderLightSerializer(serializers.ModelSerializer):
+    """Order model light serializer."""
+
+    class Meta:
+        model = models.Order
+        fields = [
+            "id",
+            "certificate",
+            "product",
```

Comment:
hmm. I see your point but it's not the default behavior of DRF. I think this can only be done on the whole project so maybe you should open an issue and see what others are thinking?